Nightscouter/SiteListTableViewController.swift

```diff
@@ -341,38 +339,26 @@ class SiteListTableViewController: UITableViewController {
         fetchSiteData(site) { (returnedSite, error: NightscoutAPIError) -> Void in
             defer {
                 print("setting networkActivityIndicatorVisible: false and stopping animation.")
-                
                 AppDataManageriOS.sharedInstance.updateSite(returnedSite)
                 UIApplication.sharedApplication().networkActivityIndicatorVisible = false
                 
                 if (self.refreshControl?.refreshing != nil) {
                     self.refreshControl?.endRefreshing()
                 }
-                
             }
             
             switch error {
             case .NoError:
-                dispatch_async(dispatch_get_main_queue(), { () -> Void in
                     self.lastUpdatedTime = returnedSite.lastConnectedDate
                     self.tableView.reloadRowsAtIndexPaths([NSIndexPath(forRow: index, inSection: 0)], withRowAnimation: .Automatic)
-                })
```

Comment:
fetchSiteData always returns on the main thread so this dispatch_async is not necessary.